pkg/core/localize/locales/en/cmd/dedicated.en.toml

```diff
@@ -26,7 +26,10 @@ one = 'The access token to use to authenticate with the OpenShift Cluster Manage
 one = 'The ID of the OpenShift cluster to deregister'
 
 [dedicated.deregisterCluster.run.noClusterFound]
-one = 'No valid OpenShift clusters found'
+one = '''
+No valid OpenShift clusters found.
+Try passing a different page number or increasing the page size.
```

Comment:
Are we asking the user to be aware of the API pagination?
I think this should remain hidden as an implementation detail and never be exposed to the user.

Reply:
This is here incase the user passes in a page via the page flag and the the cluster isn't in said page.
should this not be there then for this  case??

Reply:
oh, didn't knew we already have a "page" flag 😞